### PR TITLE
Remove 32-bit references from Hadoop

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do?

Fixes a part of #1675 

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] ~Is the TravisCI build passing?~
* [ ] ~Was the CHANGELOG.md updated to reflect the changes?~
* [ ] ~Was the documentation framework updated to reflect the changes?~

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [ ] ~Have you successfully tested your changes locally? Elaborate:~
    > ~Tested by {Running the Monkey locally with relevant config/running Island/...}~
* [ ] ~If applicable, add screenshots or log transcripts of the feature working~

### CODE REVIEW

**Review by mssalvatore — CHANGES_REQUESTED** [review]
If you follow the chain of calls all the way down, `download_monkey_exe()` Eventually ends up calling `monkey_island/cc/resources/monkey_download.py`, which still has logic regarding 32-bit agents in it. We'll have to remove that in this PR as well.

File: monkey/infection_monkey/exploit/tools/helpers.py [hunk 1]
```diff
@@ -29,11 +29,8 @@ def get_target_monkey(host):
 
     if not monkey_path:
         if host.os.get("type") == platform.system().lower():
-            # if exe not found, and we have the same arch or arch is unknown and we are 32bit,
-            # use our exe
-            if (not host.os.get("machine") and sys.maxsize < 2 ** 32) or host.os.get(
-                "machine", ""
-            ).lower() == platform.machine().lower():
+            # if exe not found, and we have the same arch, use our exe
```
Comment (VakarisZ) [inline monkey/infection_monkey/exploit/tools/helpers.py:32]: I don't understand this comment, maybe remove it?

Comment (mssalvatore) [reply monkey/infection_monkey/exploit/tools/helpers.py:32]: Looking into it.

**Review by mssalvatore — APPROVED** [review]
Approved pending a final test by @ilija-lazoroski

**Pull request merged**